app/src/main/java/fr/gaulupeau/apps/Poche/ui/preferences/SettingsActivity.java

```diff
@@ -103,6 +103,9 @@ protected void onCreate(Bundle savedInstanceState) {
 
         private boolean keepScreenOnChanged;
         private boolean oldkeepScreenOn;
+
+        private boolean swipeArticlesChanged;
```

Comment:
All these `*Changed` flags are meant for settings that need to do some action in `applyChanges()` which is not the case for this.